DPGAnalysis/MuonSysAging/plugins/RPCChamberMasker.cc

```diff
@@ -0,0 +1,224 @@
+// -*- C++ -*-
+//
+// Package:    DPGAnalysis/MuonSysAging/
+// Class:      RPCChamberMasker
+// 
+/**\class RPCChamberMasker RPCChamberMasker.cc DPGAnalysis/MuonSysAging/plugins/RPCChamberMasker.cc
+
+ Description:
+
+ Implementation:
+     Class to mask RPC digis on a for single DetIds
+
+*/
+//
+// Original Author:  Borislav Pavlov
+//         Created:  Tue, 16 Jun 2015 13:02:20 GMT
+//
+//
+
+
+// system include files
+#include <memory>
+#include <string>
+#include <vector>
+#include <iostream>
+#include <algorithm>
+#include <regex>
+
+// user include files
+#include "FWCore/Framework/interface/Frameworkfwd.h"
+#include "FWCore/Framework/interface/EDProducer.h"
+#include "FWCore/Framework/interface/ConsumesCollector.h"
+#include "FWCore/Framework/interface/ESHandle.h"
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/Framework/interface/MakerMacros.h"
+#include "FWCore/Framework/interface/EventSetup.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+
+#include "DataFormats/RPCDigi/interface/RPCDigiCollection.h"
+#include "DataFormats/MuonDetId/interface/RPCDetId.h"
+#include "CondFormats/MuonSystemAging/interface/MuonSystemAging.h"
+#include "CondFormats/DataRecord/interface/MuonSystemAgingRcd.h"
+#include "Geometry/RPCGeometry/interface/RPCGeometry.h"
+#include "Geometry/Records/interface/MuonGeometryRecord.h"
+
+#include "FWCore/ServiceRegistry/interface/Service.h"
+#include "FWCore/Utilities/interface/RandomNumberGenerator.h"
+#include "CLHEP/Random/RandomEngine.h"
+//
+// class declaration
+//
+
+class RPCChamberMasker : public edm::EDProducer 
```

Comment:
This class needs to be a modern producer (probably an `edm::stream::EDProducer`)